packages/components/psammead-social-embed/src/Canonical/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { memo } from 'react';
+import Helmet from 'react-helmet';
+import { shape, string } from 'prop-types';
+import loadable from '@loadable/component';
+
+const OEmbed = loadable(() => import('@bbc/psammead-oembed'), {
+  fallback: <p>Loading&hellip;</p>,
```

Comment:
In the future this component could take the width and height provided by the oembed to try and prevent content reflows